apiservers/portlayer/restapi/handlers/scopes_handlers.go

```diff
@@ -30,7 +30,12 @@ import (
 
 // ScopesHandlersImpl is the receiver for all of the storage handler methods
 type ScopesHandlersImpl struct {
-	ctx *network.Context
+}
+
+var ctx *network.Context
```

Comment:
I know this is old code but consider renaming this.  We've been using `ctx` for `context.Context` everywhere else.